docs/develop/contracts/js/quickstart.md

```diff
@@ -0,0 +1,328 @@
+```
+
+### Creating a project
+
+Now that you have Node, npm, and the NEAR-CLI installed, create a new directory to initialize the project in.
+
+```bash
+mkdir javascript-enclave-quickstart && cd javascript-enclave-quickstart
+```
+
+Once you're in the directory, initialize a new default npm project.
+
+```bash
+npm init -y
+```
+
+This will create a `package.json` file with contents similar to:
+
+```js
+{
+  "name": "javascript-enclave-quickstart",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC"
+}
+```
+
+The next step is to install the `near-sdk-js` package and add it as a dependency for your project. This will allow for convenient ways of building and interacting with your smart contract.
+
+```bash
+npm install --save near-sdk-js
+```
+
+Once the package has successfully been installed, you can create a convenient script in your `package.json` for building your contract. Add the following line to your `package.json` under the `scripts` section:
+
+```diff 
+{
+  "name": "javascript-enclave-quickstart",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
++   "build": "near-sdk build",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "near-sdk-js": "^0.1.1"
+  }
+}
+```
+
+> **Note:** This is optional and you can simply run `near-sdk build` instead.
+
+You'll now want to create the `src` directory and initialize a new javascript file `index.js` where your contract logic will live. 
+
+```bash
+mkdir src && cd src && touch index.js && cd ..
+```
+
+The last step is to create a new file called `babel.config.json` which allows you to configure how the contract is built. In the project root, create a new file and add the following content.
+
+```bash
+touch babel.config.json
+```
+```js
+{
+  "plugins": [
+    "near-sdk-js/src/build-tools/near-bindgen-exporter",
+    ["@babel/plugin-proposal-decorators", {"version": "legacy"}]
+  ]
+}
+```
+
+At this point, you just need to install all the packages and you will be ready to build your smart contract!
+
+```bash
+npm install
+```
+
+Your file structure should now look as follows:
+
+```
+javascript-enclave-quickstart
+├── node_modules
+├── package-lock.json
+├── babel.config.json
+├── package.json
+└── src
+    └── index.js
+```
+
+### Writing your first contract
+
+Now that you have the basic structure outlined for your project, it's time to start writing your first contract. You'll create a simple contract for setting and getting a greeting message on-chain.
+
+The contract presents 2 methods: ```set_greeting``` and ```get_greeting```. ```set_greeting```stores a String in the contract's parameter message, while ```get_greeting``` retrieves it. By default, the contract returns the message "Hello".
```

Comment:
```suggestion
The contract presents 2 methods: `set_greeting` and `get_greeting`: 
- `set_greeting```stores a String in the contract's parameter message, 
- while `get_greeting` retrieves it. 

By default, the contract returns the message "Hello".
```